pandas/util/testing.py

```diff
@@ -1215,6 +1215,10 @@ def assert_frame_equal(left, right, check_dtype=True,
                        obj='DataFrame'):
     """Check that left and right DataFrame are equal.
 
+    Each element at [row, column] in left is compared to the
+    element at [row, column] in right (index and column values
```

Comment:
this last part of the comment adds to confusion. can you clarify. Maybe add this to a Notes section

Reply:
I moved the clarification into the parameter description.